OCCommunicationLib/OCCommunicationLib/OCSharedDto.h

```diff
@@ -56,6 +56,10 @@ typedef enum {
 @property (nonatomic, copy) NSString *displayNameOwner;
 @property (nonatomic, copy) NSString *uidFileOwner;
 @property (nonatomic, copy) NSString *fileTarget;
+@property (nonatomic, copy) NSString *name;
+@property (nonatomic, copy) NSString *url;
 
 
+- (id)initWithSharedDto:(OCSharedDto *)oSharedDto;
```

Comment:
Name should be ```OCShareDto```, without the final ```d```; and the same for ```initSharedDto``` -> ```initShareDto``` . The conceptual resource represented is a Share.